.eslintrc.json

```diff
@@ -39,6 +42,7 @@
         "requirejs/no-multiple-define": 2,
         "requirejs/no-named-define": "off",
         "requirejs/no-commonjs-wrapper": 2,
-        "requirejs/no-object-define": 1
+        "requirejs/no-object-define": 1,
+        "react/prop-types": "off"
     }
 }
```

Comment:
Suggest adding the following to remove a warning:
```json
"settings": {
    "react": {
        "version": "16.14"
    }
}
```

Reply:
Note: this will be `"version": "17.02"` once https://github.com/adaptlearning/adapt_framework/pull/3153 is merged.